main.go

```diff
@@ -18,5 +23,9 @@ func main() {
 		log.Fatalf("DISCORD_BOT_TOKEN must be set")
 	}
 	AutoLoadSave = !(os.Getenv("GROL_DISABLE_AUTOSAVE") == "1")
+	memlimit := debug.SetMemoryLimit(-1)
+	if memlimit == math.MaxInt64 {
+		log.Fatalf("Memory limit not set, please set GOMEMLIMIT=1GiB or similar")
```

Comment:
Same comment about adding information it's about an env variable

Reply:
yeah but I figured the only operator of the bot which is me knows what GOMEMLIMIT= implies :)